angelsrefining/prototypes/map-gen-presets.lua

```diff
@@ -0,0 +1,260 @@
+
+data.raw['map-gen-presets'].default.default.default = false
+
+--set starting area size
+data.raw['map-gen-presets'].default.default.basic_settings = {}
+
+--data.raw['map-gen-presets'].default.basic_settings.starting_area = 'large'
+
+--remove pollution
+data.raw['map-gen-presets'].default.default.advanced_settings = {}
+
+data.raw['map-gen-presets'].default.default.advanced_settings =
+  {
+    pollution =
+      {
+        enabled = false
+      },
+      order = 'a'
+  }
+
+--remove biters
+data.raw['map-gen-presets'].default.default.basic_settings =
+  {
+    autoplace_controls =
+      {
+        ["enemy-base"] = { frequency = "none", size = "none"}
+      },
+      starting_area = 'big',
+      order = 'a'
+  }
+
+--[[
```

Comment:
This can be removed, right?

Reply:
We decided to leave the defaults in as a lookup